dbms/src/Storages/DeltaMerge/StoragePool.cpp

```diff
@@ -230,7 +314,22 @@ bool StoragePool::gc(const Settings & settings, const Seconds & try_gc_period)
         last_try_gc_time = now;
     }
 
-    return doStoragePoolGC(global_context, settings, *this);
+    // Only do the v2 GC
```

Comment:
Can we wrap those into a separate method?

Reply:
I don't understand. why do we need split v2 GC?

Reply:
I mean putting those codes into a separate method, like gc_v2, and then calling it in the method. But it is a minor suggestion, up to you.

Reply:
done